scripts/benchmarks_local.py

```diff
@@ -62,6 +62,18 @@ class RunType(Enum):
 def is_windows(parsed_args: Namespace):
     return parsed_args.os == "windows"
 
+def get_os_short_name(os_name: str):
+    if os_name == "windows":
+        return "win"
+    elif os_name == "linux":
+        return "linux"
+    elif os_name == "osx":
```

Comment:
Please check if it is darwin or osx.

Reply:
We currently normalize on OSX for Darwin systems: https://github.com/dotnet/performance/blob/0a028adbaced3b7db7d3efa9792c748f98513079/scripts/benchmarks_local.py#L572. If you know that the runtime build paths end up using darwin or osx specifically I can update to use whichever/both, otherwise I would like to stay normalized on osx for consistency.

Reply:
Thanks. All good.